deployment/ansible/roles/ckan-odp-configuration/templates/backup.sh.j2

```diff
@@ -14,5 +17,6 @@ pushd $CKAN_DIR
 
 $PASTER db dump --config=$CKAN_CONFIG $BACKUP_FILE
 gzip -f -9 $BACKUP_FILE
+s3cmd -c $S3CMD_CFG put $BACKUP_FILE.gz s3://$S3_BUCKET/$DATE-$HOSTNAME.gz
```

Comment:
Maybe I'm missing something, but I don't see an explicit installation step for `s3cmd` in any of the Ansible playbooks. If there's not one it would be good to add it, even if it's getting installed as a side effect of something else.